src/controllers/challenge.controller.js

```diff
@@ -1,17 +1,16 @@
 // Mock repository
 const data = require("../../ex.json");
 
-const getRandomChallenge = (req, res) => {
-  const question = data[Math.floor(Math.random() * data.length)];
-  res.send(question);
-};
-
-const getChallengeById = (req, res) => {
-  const question = data.find((obj) => obj.id === req.params.id);
-  res.send(question);
+const getChallenge = (req, res) => {
+  let options = [...data];
```

Comment:
I'm not sure on the variable name options here.

You're also mutating an array.